content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -476,18 +475,19 @@ account can be passed into your program's instructions, so you should verify
 it’s the account you expect it to be.
 
 In Anchor, if you deserialize the account to the `AggregatorAccountData` type
-from the `switchboard_v2` crate, Anchor checks that the account is owned by the
-Switchboard program. If your program expects that only a specific data feed will
-be passed in the instruction, then you can also verify that the public key of
-the account passed in matches what it should be. One way to do this is to hard
-code the address in the program somewhere and use account constraints to verify
-the address passed in matches what is expected.
+from the `switchboard_solana` crate, Anchor checks that the account is owned by
+the Switchboard program. If your program expects that only a specific data feed
+will be passed in the instruction, then you can also verify that the public key
+of the account passed in matches what it should be. One way to do this is to
+hard code the address in the program somewhere and use
+[account constraints](https://www.anchor-lang.com/docs/account-constraints) to
+verify the address passed in matches what is expected.
 
 ```rust
 use {
   anchor_lang::prelude::*,
   solana_program::{pubkey, pubkey::Pubkey},
-	switchboard_v2::{AggregatorAccountData},
+	switchboard_solana::{AggregatorAccountData},
```

Comment:
Can you also format the rust (4 spaces is what rustfmt uses) for this and the other repo? See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#rust--anchor

Reply:
sure